backintime-demo/app/build.gradle.kts

```diff
@@ -58,6 +58,7 @@ android {
 dependencies {
     implementation(project(":backintime-library:runtime"))
     implementation(project(":backintime-library:annotations"))
+    implementation(project(":backintime-library:websocket:event"))
```

Comment:
required because the compiler plugin uses `PropertyInfo` class when applying to compilation.